storybook/stories/staking/CountdownParty.stories.js

```diff
@@ -4,18 +4,21 @@ import { storiesOf } from '@storybook/react';
 import { withKnobs, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 
+import StoryDecorator from '../_support/StoryDecorator';
 import TopBar from '../../../source/renderer/app/components/layout/TopBar';
 import StakingInfoCountdown from '../../../source/renderer/app/components/staking/info/StakingInfoCountdown';
 import NodeSyncStatusIcon from '../../../source/renderer/app/components/widgets/NodeSyncStatusIcon';
 import NewsFeedIcon from '../../../source/renderer/app/components/widgets/NewsFeedIcon';
 import TadaButton from '../../../source/renderer/app/components/widgets/TadaButton';
 
-storiesOf('Decentralization | Countdown', module)
-  .addDecorator(withKnobs)
+const decorators = [
```

Comment:
Decorators weren't unmounted within the same story group thus rendering duplicated UI elements on the next story.